geode-core/src/main/java/org/apache/geode/management/internal/cli/shell/jline/ANSIHandler.java

```diff
@@ -33,10 +30,7 @@ public ANSIHandler(boolean isAnsiEnabled) {
   }
 
   public static ANSIHandler getInstance(boolean isAnsiSupported) {
-    if (instance == null) {
-      instance = new ANSIHandler(isAnsiSupported);
-    }
-    return instance;
+    return new ANSIHandler(isAnsiSupported);
```

Comment:
is there any good reason this would have been a singleton before?  Are ANSIHandlers expensive to create or something?

Reply:
I don't think there is a good reason. It's a small class with one boolean field for state. Really cheap.